samples/app/src/main/kotlin/com/bumble/appyx/app/node/onboarding/OnboardingContainerNode.kt

```diff
@@ -169,12 +154,36 @@ class OnboardingContainerNode(
                                 color = appyx_dark,
                             )
                         }
-                        Spacer(Modifier)
                     }
                 }
             }
         }
     }
+
+    @Composable
+    private fun PreviousAndNextButtons(
```

Comment:
I believe this change might go against the Compose Lint rules that we will likely introduce in the future. The problem is that there are now multiple composables being created without a specified scope (Column/Row), so calling this could have unindented behaviour

Reply:
I guess we can resolve that once we introduce the lint rules.

Reply:
I'm not familiar with that Compose Lint rule, you mean just to do `private fun RowScope.PreviousAndNextButtons`?
If that's the case I can do it now